public/app/features/dashboard-scene/panel-edit/PanelEditorRenderer.tsx

```diff
@@ -38,6 +43,25 @@ export function PanelEditorRenderer({ model }: SceneComponentProps<PanelEditor>)
 
   return (
     <>
+      {isScrollingLayout && (
+        <Global
```

Comment:
This feels very hacky , to have hook that adds a global css style that overrides global styles.

This should be part of those global styles via plain media queries